website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,193 @@
+description: "Use exports to materialize tables to the data platform on a schedule."
+sidebar_label: "Materialize with exports"
+---
+
+The exports feature in dbt Semantic Layer enhances the [saved queries](/docs/build/saved-queries) by allowing you to materialize commonly used queries directly within your data platform.
+
+While saved queries are a way to save and reuse commonly used queries in MetricFlow, exports take this functionality a step further by:
+
+- Enabling you to materialize these queries within your data platform using dbt Cloud.
+- Proving an integration path for tools that don't natively support the dbt Semantic Layer by exposing tables of metrics and dimensions.
+
+Essentially, exports are like any other table in your data platform. They enable you to query metric definition through any SQL interface or connect to downstream tools without needing a first class Semantic Layer integration. Refer to [Available integrations](/docs/use-dbt-semantic-layer/avail-sl-integrations) for more information.
+
+## Prerequisites
+
+- You have a dbt Cloud account on a [Team or Enterprise](https://www.getdbt.com/pricing/) plan.
+- You are on [dbt version](/docs/dbt-versions/upgrade-core-in-cloud) 1.7 or newer.
+- You have the dbt Semantic Layer [configured](/docs/use-dbt-semantic-layer/setup-sl) in your dbt project.
+- You have a dbt Cloud environment with a [Job scheduler](/docs/deploy/job-scheduler) enabled.
+
+### Comparison between exports and saved queries
+
+
+| Feature |  Exports | <div style={{width:'250px, text-align: center'}}>Saved queries</div>  |
+| ----------- | ----------- | ---------------- |
+| **Availability**    | Available on dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing/) plans with dbt versions 1.7 or newer.| Available in both dbt Core and dbt Cloud.     |
+| **Purpose**         | To materialize saved queries in your data platform and expose metrics and dimensions as a view or table. | To define and manage common Semantic Layer queries in YAML, including metrics and dimensions.   |
+| **Usage**           | Automatically runs saved queries and materializes them within your data platform. Exports count towards [queried metrics](/docs/cloud/billing#what-counts-as-a-queried-metric) usage. <br /><br />Example: Create a weekly aggregated table for active user metrics, automatically updated and stored in the data platform.  | Used for organizing and reusing common MetricFlow queries within dbt projects.<br /><br /><br />Example: Group related metrics together for better organization, and include commonly uses dimensions and filters. | For materializing query results in the data platform. |
+| **Integration**     | Must have the dbt Semantic Layer configured in your dbt project.<br /><br />Tightly integrated with the [MetricFlow Server](/docs/use-dbt-semantic-layer/sl-architecture#components) and dbt Cloud's Job scheduler. | Integrated into the dbt <Term id="dag" /> and managed alongside other dbt nodes. |
```

Comment:
missed this yesterday, sorry! 

similar to "exports" & "saved queries" , it's also "job scheduler" as it's a feature (vs a product)

```suggestion
| **Integration**     | Must have the dbt Semantic Layer configured in your dbt project.<br /><br />Tightly integrated with the [MetricFlow Server](/docs/use-dbt-semantic-layer/sl-architecture#components) and dbt Cloud's job scheduler. | Integrated into the dbt <Term id="dag" /> and managed alongside other dbt nodes. |
```